spec.html

```diff
@@ -120,6 +120,32 @@ <h1>
 		<emu-note type=editor>
 			In the case of an abrupt ~throw~ completion, the type of error to be created should match the type of the abrupt throw completion record. This could be revisited when merging into the main specification. Additionally, in the case of a ~break~ or ~continue~ completion, since those are not supported, a TypeError is expected.
 		</emu-note>
+		<emu-note>
+			Host environments may provide implementation-specific stack information for introspection APIs. If so, all Error objects originating from code executed inside a ShadowRealm must be subject to censorship. The specific steps are implementation-defined but must respect the following: The information exposed as part of Error's stack must contain only the details of the stack-frames associated to the ShadowRealm, and not reveal any details of frames associated with other realms, including the number of any such frames. Implementations can store information of the full stack if the error can cross into another Realm, however the implementation cannot use that information to reveal more information than allowed above.
+		</emu-note>
+	</emu-clause>
+
+	<emu-clause id="sec-create-type-error-copy" type="abstract operation">
+		<h1>
+			CreateTypeErrorCopy (
+				_realmRecord_: a Realm Record,
+				_originalError_: an ECMAScript language value
+			): A TypeError object
+		</h1>
+		<emu-alg>
+			1. Let _newError_ be a newly created *TypeError* object.
+			1. NOTE: _newError_ is created in _realmRecord_.
+			1. If Type(_originalError_) is Object, then
+				1. Let _message_ be Completion(Get(_originalError_, "message")).
+				1. If _message_ is not an abrupt completion, then
+					1. If Type(_message_) is String, then
+						1. NOTE: Copying the string message when possible.
+						1. Perform CreateNonEnumerableDataPropertyOrThrow(_newError_, "message", _message_).
+			1. Return _newError_.
+		</emu-alg>
+		<emu-note>
+			Host environments may provide implementation-specific stack information for introspection APIs. The implementation may augment the exposed stack information of _newError_ with information it has stored on _originalError_. If _realmRecord_ corresponds to a ShadowRealm, _newError_ object must be subject to censorship in the same way Errors originated from _realmRecord_ are censored. Regardless of the _realmRecord_, the implementation must not include object references to _originalError_ or its associated realm in the stack information exposed to the program.
```

Comment:
@mhofman @littledan check this note... and the one above...

Reply:
This also should be simply a paragraph and not a note.